whitepapers/Viacoin_whitepaper.tex

```diff
@@ -135,17 +135,17 @@ \section{The Lightning Network}\label{sec: The Lightning Network}
 become a focal point for routing the payment of others leading into a fully
 connected payment channel. The payments are enforced using a script which enforces the atomicity via decrementing time-locks.
 \newline \newline \noindent
-Another benefit is the possibility of atomic cross-chain transactions, enabling users to trade viacoin, bitcoin, litecoin and other Segwit coins
+Another benefit is the possibility of atomic cross-chain transactions, enabling users to trade Viacoin, Bitcoin, Litecoin and other Segwit coins
```

Comment:
Bitcoin refers to the Bitcoin protocol
bitcoin refers to the currency itself.

I think they should stay lowercase

Reply:
ah fair enough, sounds good